docker-compose.yml

```diff
@@ -2,16 +2,16 @@ version: "3.2"
 
 services:
   dev:
-    container_name: node-template
+    container_name: mintlayer-core
     image: paritytech/ci-linux:974ba3ac-20201006
-    working_dir: /var/www/node-template
+    working_dir: /var/www/mintlayer-core
     ports:
       - "9944:9944"
     environment:
-      - CARGO_HOME=/var/www/node-template/.cargo
+      - CARGO_HOME=/var/www/mintlayer-core/.cargo
     volumes:
-      - .:/var/www/node-template
+      - .:/var/www/mintlayer-core
       - type: bind
         source: ./.local
         target: /root/.local
-    command: bash -c "cargo build --release && ./target/release/node-template --dev --ws-external"
+    command: bash -c "cargo build --release && ./target/release/mintlayer-core --dev --ws-external"
```

Comment:
I have not tested the docker changes here, it is just a blind search/replace. Is anybody using this?

Reply:
I don't usually use it but I can give it a quick spin as I have played with it before

Reply:
Seems fine to me

Reply:
Somewhat unrelated but at some point we probably should provide a Dockerfile that installs everything needed and launches the node. Right now a lot of people seem to have problems connecting to our testnet but if all they had to do was run `docker run -t mintlayer-core`, I'm sure the number of support requests would drop

Reply:
Yeah dealing with docker support requests will be much more fun :smiley: